pints/_abc/_abc_smc.py

```diff
@@ -0,0 +1,176 @@
+#
+# ABC SMC method
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class ABCSMC(pints.ABCSampler):
+    """
+    ABC-SMC Algorithm  See, for example, [1]_. In each iteration of the
```

Comment:
The first paragraph of every docstring should be a single-line, stand-alone explanation. So can't refer to things, can't be 2 sentences etc.
The reason for this is that IDEs (and some python help functions) will show the first line of docstrings to help the user remember what everything is.